src/styles/components/_input.scss

```diff
@@ -5,7 +5,7 @@ $component-name: 'input';
   margin: 20px 0 0;
 
   &__field {
-    @include text(h2);
+    @include type('title-2');
```

Comment:
type as in type or type as in typography? If the latter can we remove the abbreviation?